repo/packages/E/elasticsearch/1/marathon.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "{{elasticsearch.framework-name}}",
+  "cpus": {{elasticsearch.scheduler.cpu}},
+  "mem": {{elasticsearch.scheduler.ram}},
+  "ports": [0],
+  "instances": {{elasticsearch.scheduler.instances}},
+  "healthChecks": [
+    {
+      "gracePeriodSeconds": 120,
+      "intervalSeconds": 30,
+      "maxConsecutiveFailures": 0,
+      "path": "/",
+      "portIndex": 0,
+      "protocol": "HTTP",
+      "timeoutSeconds": 5
+    }
+  ],
+  "args": [
+    "--zookeeperUrl={{mesos.master}}",
+    "--zookeeperTimeout={{elasticsearch.zookeeper-timeout}}",
+    "--webUiPort={{elasticsearch.port}}",
+    "--frameworkFailoverTimeout={{elasticsearch.failover-timeout}}",
+    "--frameworkName={{elasticsearch.framework-name}}",
+
+    "--elasticsearchNodes={{elasticsearch.executor.instances}}",
+    "--elasticsearchClusterName={{elasticsearch.executor.cluster-name}}",
+    "--elasticsearchCpu={{elasticsearch.executor.cpu}}",
+    "--elasticsearchDisk={{elasticsearch.executor.disk}}",
+    "--elasticsearchRam={{elasticsearch.executor.ram}}",
+    "--elasticsearchSettingsLocation={{elasticsearch.executor.settings-location}}",
+
+    "--executorForcePullImage={{elasticsearch.force-pull-image}}",
+    "--executorHealthDelay={{elasticsearch.executor.health-delay}}",
+    "--executorImage={{elasticsearch.executor.image}}",
+    "--executorName={{elasticsearch.executor.name}}",
+    "--executorTimeout={{elasticsearch.executor.timeout}}",
+  ],
+  "env": {
+    "JAVA_OPTS": "{{elasticsearch.scheduler.java-heap}}"
+  },
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mesos/elasticsearch-scheduler",
```

Comment:
Make this a parameter, and set the default to "mesos/elasticsearch-scheduler:0.3.0"